### PR TITLE
Add a context size change test.

### PR DESCRIPTION
**Pull request by pavolas**
Add a context size change test.

The test makes an antialiased, no alpha context. Then it resizes it, and checks
that clearing/reading the resized context still works.

### CODE REVIEW

File: sdk/tests/conformance/context/00_test_list.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 --max-version 1.9.9 constants-and-properties.html
 --min-version 1.0.2 context-attribute-preserve-drawing-buffer.html
 context-attributes-alpha-depth-stencil-antialias.html
+context-size-change.html
```
Comment (kenrussell) [inline sdk/tests/conformance/context/00_test_list.txt]: Please add --min-version 1.0.4 as a prefix (it's just the way new tests are added at this point).

**Comment by kenrussell** [comment]
Thanks, looks good. Could you please make the one tiny fix above?

**Comment by pavolas** [comment]
Done.

**Comment by kenrussell** [comment]
Excellent. Thanks for adding this test.

**Pull request merged**